setup.py

```diff
@@ -45,7 +45,7 @@ class PyPackage:
                   "bitsandbytes", "safetensors", "peft", "triton",
                   "torchinfo", "mpi4py", "deepspeed>=0.15.3",
                   "sentence-transformers", "huggingface-hub>=0.26.2",
-                  "aiohttp", "setproctitle"],
+                  "aiohttp", "setproctitle", "google-auth>=2.37.0"],
```

Comment:
google-auth? for what purpose?

Reply:
litellm requires it to refer to google api